pyproject.toml

```diff
@@ -1,6 +1,7 @@
 [tool.poetry]
 name = "aladdin"
-version = "1.15.6.16"
```

Comment:
this one didn't get bumped, not sure if there's an easy way to keep the two in sync, I added a code comment for now